Monitron.Node/NodeConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Monitron.Common;
+using System.Xml;
+
+namespace Monitron.Node
+{
+	public sealed class NodeConfiguration
+	{
+		public Account Account { get; private set;}
+		public string Plugin { get ; private set;}
+		public string PluginAssemblyName { get; private set;}
+		//TBD - add plugin configuration
+
+		public NodeConfiguration(){
+		}
+			
+		public void LoadConfigFromFile(string i_FilePath)
+		{
+			XmlDocument doc = getXmlDocument (i_FilePath);
+			String userName = doc.GetElementsByTagName ("Username")[0].InnerXml;
+			String domain = doc.GetElementsByTagName ("Domain")[0].InnerXml;
+			String password = doc.GetElementsByTagName ("Password")[0].InnerXml;
+			Plugin = doc.GetElementsByTagName ("Type")[0].InnerXml;
+			PluginAssemblyName = doc.GetElementsByTagName ("AssemblyName")[0].InnerXml;
+			Account = new Account (userName, password, domain);

```

Comment:
Look at the IXMLSerializable interface.

https://msdn.microsoft.com/en-us/library/system.xml.serialization.ixmlserializable(v=vs.110).aspx